addons/web/static/src/js/core/utils.js

```diff
@@ -737,6 +737,17 @@ var utils = {
         }
         return curr;
     },
+    /**
+     * return the domain targeting assets files
+     *
+     * @returns {Array} Domain of assets files
+     */
+    assetsDomain: function () {
+        return [
+            ['res_model', '=', 'ir.ui.view'],
+            ['name', 'like', 'assets_']
+        ];
+    }
```

Comment:
I am being strict but:
- "Returns the domain targeting assets files." (Uppercase, ending point, conjugated verb)
- Trailing comma
- I know it was there before, but as you touch the code around: there is an useless blank line after it remove it :)

Reply:
You forgot about the trailing comma.

Reply:
I think you are looking at the wrong version I did add a comma there.

Reply:
`},` you added the comma in the array which I don't think we do actually